.github/workflows/CI.yml

```diff
@@ -25,9 +25,10 @@ jobs:
       run: npm ci --legacy-peer-deps
     - name: Lint
       run: npm run lint
-    # Lint changes should be pushed
-    # to the branch before the branch
-    # is merge eligible.
+    - name: Spellcheck
+      run: npm run spellcheck
```

Comment:
Do we have the usage tips from the PR description documented anywhere that's more easily findable? :eyes:

Reply:
We did not but that is a great idea so I added it to contributing: https://github.com/ionic-team/ionic-docs/pull/3069/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055

You can preview the file here: https://github.com/ionic-team/ionic-docs/blob/style-add-spellcheck/CONTRIBUTING.md#development-workflow

Please pick this apart as I wasn't sure about some of the formatting / verbiage!

Reply:
Looks great to me! Thank you :D